src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitor.java

```diff
@@ -97,4 +104,20 @@ public void stopMonitor() {
             monitorFuture.cancel(true);
         }
     }
+
+    /**
+     * Remove cert from cert expiry monitor.
+     * @param cg CertificateGenerator instance for the certificate
+     */
+    public synchronized void removeFromMonitor(CertificateGenerator cg) {
```

Comment:
Note: adding/removing cert generators from a PQ is just asking for concurrency issues. Let's just add CGs to a CopyOnWriteSet and let the monitor loop check iterate through them all. There will realistically only be 1-2 CGs in it so it's probably cheaper anyway and avoids this nonsense